src/test/java/org/junitpioneer/jupiter/ReportEntryExtensionTest.java

```diff
@@ -83,26 +83,18 @@ void repeatedAnnotation_logEachKeyValuePairAsIndividualEntry() {
 						"rapping at my chamber door"));
 	}
 
-	private static List<Map<String, String>> reportEntries(ExecutionEventRecorder recorder) {
-		return recorder
-				.eventStream()
-				.filter(event -> event.getType().equals(ExecutionEvent.Type.REPORTING_ENTRY_PUBLISHED))
-				.map(executionEvent -> executionEvent.getPayload(org.junit.platform.engine.reporting.ReportEntry.class))
-				.filter(Optional::isPresent)
-				.map(Optional::get)
-				.map(org.junit.platform.engine.reporting.ReportEntry::getKeyValuePairs)
-				.collect(toList());
-	}
-
 	private static Map.Entry<String, String> entryOf(String key, String value) {
 		return new AbstractMap.SimpleEntry<>(key, value);
 	}
 
 	static class ReportEntriesTest {
 
+		private static AtomicInteger executionCount = new AtomicInteger(0);
+
 		@Test
 		@ReportEntry(key = "Crow2", value = "While I pondered weak and weary")
 		void explicitKey() {
+			System.out.println("HI THERE! " + executionCount.getAndIncrement());
```

Comment:
Why is this here?

Reply:
Nicolai added it for basic tests after creating the basic wrapper after updating so see if it works. And I just didn't think about it anymore. Will remove it